### ISSUE TITLE
bar background color in single knob sliders

### ISSUE THREAD

**Issue by ghost** [Venturocket/angular-slider#44]
Hello,

Thanks for this great module!

When using single knob sliders, is it possible to give the bar a different background color from the floor point until the selection point? (i.e., if the slider goes from 0 to 1 and the knob is in 0.7, I want the bar to be red from 0 to 0.7 and grey from 0.7 to 1).

I've been playing with the CSS, but I didn't manage to get it working.

Thanks!

**Comment by Vitenok** [comment]
Yes, it would be great to have. Looks a bit weird not having such a basic natural functionality especially when dual knob slider allows you to do this. Trying to hack issue by making single knob slider from dual knob, so the result will be something like in JQueryUI: http://jqueryui.com/slider/#rangemin.

Have someone tried to do it this way?

Thanks in advance!

**Comment by chedched** [comment]
I'd love to see this as well. Any progress on this @Vitenok ? But @drgould mentioned he's re-writing the whole thing anyway, so maybe it makes more sense to wait for that.
